scripts/breed_bird_survey.py

```diff
@@ -16,7 +16,7 @@
 from retriever.lib.templates import Script
 from retriever.lib.models import Table, Cleanup, correct_invalid_value
 try:
-    from retriever.lib.scripts import open_fr, open_fw
+    from retriever.lib.tools import open_fr, open_fw
```

Comment:
Is this change going to break this script on 2.1.0? If so this issue would also apply to all Python scripts.